sim/dex-species.ts

```diff
@@ -136,6 +136,8 @@ export class Species extends BasicEffect implements Readonly<BasicEffect & Speci
 	declare readonly evoItem?: string;
 	/** Evolution move. falsy if doesn't evolve. */
 	readonly evoMove?: string;
+	/** Evolution region. falsy if doesn't evolve. */
```

Comment:
I would make this clear that this is _exclusive_ evo region, i.e. it cannot evolve anywhere else, even if it might be obtainable elsewhere.